airgun/entities/subscription.py

```diff
@@ -107,6 +107,18 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def filter_columns(self, columns=None):
+        """Filters column headers
+        :param columns: dict mapping column name to boolean value
+        :return: tuple of the name of the headers
+        """
+        view = self.navigate_to(self, 'All')
+        if columns is not None:
+            view.columns_filter_checkboxes.fill(columns)
+            self.browser.plugin.ensure_page_safe()
+            view = self.navigate_to(self, 'All')
```

Comment:
sorry, but we cannot submit code without exact reason, even if that block something. What issue do you have here? `StaleElementException`? We can look on alternative solutions. E.g. waiter that will check for necessary element is visible.

Reply:
@oshtaier yes, I encountered the `StaleElementReferenceException` here.  I've tried several solutions that was suggested while working with @ldjebran on this piece and this was the only one that worked well enough during the time.